package.json

```diff
@@ -1,65 +1,58 @@
 {
-  "name": "jurors-microsite",
+  "name": "anj.aragon.org",
   "version": "1.0.0",
-  "main": "src/index.js",
-  "homepage": "https://aragon.github.io/jurors-microsite",
+  "private": true,
+  "homepage": "https://anj.aragon.org/",
   "license": "AGPL-3.0-or-later",
   "scripts": {
-    "stage": "react-static build --staging",
-    "start": "NODE_PRESERVE_SYMLINKS=1 react-static start",
-    "start:mainnet": "CHAIN_ID=1 npm start",
-    "start:rinkeby": "CHAIN_ID=4 npm start",
-    "build": "NODE_PRESERVE_SYMLINKS=1 react-static build",
+    "build": "next build",
     "build:mainnet": "CHAIN_ID=1 npm run build",
     "build:rinkeby": "CHAIN_ID=4 npm run build",
-    "analyze": "react-static build --analyze",
-    "serve": "serve dist -p 3000",
-    "deploy": "NODE_ENV=production SITE_ROOT=https://anj.aragon.org/ npm run build && echo 'anj.aragon.org' > dist/CNAME && gh-pages -d dist",
+    "deploy": "NODE_ENV=production npm run build && echo 'anj.aragon.org' > dist/CNAME && gh-pages -d dist",
     "deploy:mainnet": "CHAIN_ID=1 npm run deploy",
     "deploy:rinkeby": "CHAIN_ID=4 npm run deploy",
```

Comment:
I wonder if we should remove the deploy scripts, since they don't capture the whole story anymore (e.g. API keys, etc).

If we need to deploy without the automated action at any time, looking into the workflow file should be enough documentation :).

Reply:
Agreed, done in https://github.com/aragon/anj.aragon.org/pull/23/commits/c88450ef879d47a01d76943ec0c25fa83980086f